app/internal/actor/attorney.go

```diff
@@ -4,18 +4,25 @@ import (
 	"fmt"
 	"strings"
 
-	"github.com/ministryofjustice/opg-modernising-lpa/internal/date"
-	"github.com/ministryofjustice/opg-modernising-lpa/internal/place"
+	"github.com/ministryofjustice/opg-modernising-lpa/app/internal/date"
+	"github.com/ministryofjustice/opg-modernising-lpa/app/internal/place"
 	"golang.org/x/exp/slices"
 )
 
+// Attorney contains details about an attorney or replacement attorney, provided by the applicant
```

Comment:
this is a small thing, but generally structs would start with "A" or "An" and the fields/methods would start with the field or method name, e.g. https://pkg.go.dev/net/http#Client